src/commands/gist.ts

```diff
@@ -697,4 +751,8 @@ export async function registerGistCommands(context: ExtensionContext) {
       }
     )
   );
+  
+  context.subscriptions.push(
+    commands.registerCommand(`${EXTENSION_NAME}.syncGist`, syncGistInternal)
```

Comment:
This should use `registerTextEditorCommand` since it's expected to be called when there's an active editor open. Then the handler would receive the `TextEditor` instance as an argument.